e2e/suites/actions/upload-download/upload-file.test.ts

```diff
@@ -50,19 +50,73 @@ describe('Upload files', () => {
+    await page.sidenav.menu.uploadFilesInput.sendKeys(`${__dirname}/upload-file.test.ts`);
+    await page.sidenav.closeNewMenu();
+    await page.uploadFilesDialog.uploadDialog.isVisible();
     done();
   });
 
-  afterAll(async (done) => {
+  afterAll(async () => {
     await apis.user.nodes.deleteNodeById(folder1Id);
-    done();
   });
 
   it('Upload a file', async () => {
-    await dataTable.doubleClickOnRowByName(folder1);
-    await page.sidenav.openNewMenu();
-    await page.sidenav.menu.uploadFilesInput.sendKeys(`${__dirname}/upload-file.test.ts`);
+    const uploadedFiles = await dataTable.isItemPresent('upload-file.test.ts');
+    expect(uploadedFiles).toBe(true, 'file not uploaded');
+  });
+
+  it('[T14752064] Close the upload dialog ', async () => {
+    await page.uploadFilesDialog.closeUploadButton.click();
+    await page.uploadFilesDialog.uploadDialog.isPresent();
+    await expect(await page.uploadFilesDialog.uploadDialog.isVisible()).toBe(false);
+  });
+
+  it('[T14752051] Minimize / maximize the upload dialog ', async () => {
+    await page.uploadFilesDialog.minimizeButton.click();
+    await expect(await page.uploadFilesDialog.uploadedFiles.waitNotVisible()).toBe(true);
+
+    await page.uploadFilesDialog.maximizeButton.click();
+    await expect(await page.uploadFilesDialog.uploadedFiles.waitVisible()).toBe(true);
+  });
+
+  describe('[T14752053]', () => {
+    it('Upload history is expunged on browser refresh ', async () => {
+      await page.refresh();
+      const isUploadDialogVisible = await page.uploadFilesDialog.uploadDialog.isVisible();
+
+      await expect(isUploadDialogVisible).toBe(false);
+    });
+
+    it('Upload history is expunged on browser login/logout ', async () => {
+      await page.signOut();
+      await loginPage.loginWith(username);
+      const isUploadDialogVisible = await page.uploadFilesDialog.uploadDialog.isVisible();
+
+      await expect(isUploadDialogVisible).toBe(false);
+    });
+  });
+
+  it('[T14752052] Upload dialog remains fixed in the browser when user performs other actions in parallel ', async () => {
+    await expect(page.uploadFilesDialog.uploadDialog.isVisible()).toBe(true);
+
+    await page.clickPersonalFiles();
+    await expect(page.uploadFilesDialog.uploadDialog.isVisible()).toBe(true);
+
+    await page.clickFileLibraries();
+    await expect(page.uploadFilesDialog.uploadDialog.isVisible()).toBe(true);
+
+    await page.clickSharedFiles();
+    await expect(page.uploadFilesDialog.uploadDialog.isVisible()).toBe(true);
+
+    await page.clickRecentFiles();
+    await expect(page.uploadFilesDialog.uploadDialog.isVisible()).toBe(true);
+
+    await page.clickFavorites();
+    await expect(page.uploadFilesDialog.uploadDialog.isVisible()).toBe(true);
 
-    expect(await dataTable.isItemPresent('upload-file.test.ts')).toBe(true, 'file not uploaded');
+    await page.clickTrash();
+    await expect(page.uploadFilesDialog.uploadDialog.isVisible()).toBe(true);
```

Comment:
I think, that w can check this thing in approximately 2 different cases. You don't need to check it by clicking in each link and expecting that dialog is still visible. Please change this code and leave only this:
```
await expect(page.uploadFilesDialog.uploadDialog.isVisible()).toBe(true);
await page.clickPersonalFiles();
await expect(page.uploadFilesDialog.uploadDialog.isVisible()).toBe(true);
```

Reply:
I have changed it. I was following the manual regression test case and it was checked from these places